http-endpoint-example/README.md

```diff
@@ -0,0 +1,46 @@
+# How to download Met Office atmospheric model data using an HTTP endpoint
+A demonstration code example of downloading objects in response to SNS messages via an HTTP endpoint.
+
+By using the following you will be able to subscribe to an Amazon Web Services (AWS) Simple Notification Service (SNS) topic and start downloading objects.
+The running python script will accept notifications from the SNS topic and download the objects according to the input parameters.
+
+
+## Prerequisites:
+
+  * A service API key
+  * A Python 3 environment containing the flask and requests library.
```

Comment:
"libraries", not "library"
Also, none of the other bullet points finishes with a full stop, so this shouldn't, either